src/DynamoDbModel.php

```diff
@@ -195,10 +218,10 @@ public static function all($columns = [], $limit = -1)
         return $model->getAll($columns, $limit);
     }
 
-    public static function first($columns = [])
+    public function first($columns = [])
```

Comment:
Yes to the test case you mention.

Let's go thru the cases that we usually have for Eloquent:
```
$userQuery = User::where('name', 'foo');
$roleQuery = Role::where('name', 'bar');

// then $userQuery and $roleQuery should still have their conditions remained intact.
$userQuery->first();
$roleQuery->first();
```

I feel like we need to think of a better way. Using this `static::$instance` isn't reliable just as you mentioned above.